apps/web/src/server/applications/case/examination-timetable/applications-timetable.controller.js

```diff
@@ -80,7 +80,7 @@ export const uniqueTimeTableTypes = {
 export async function viewApplicationsCaseTimetableList(_, response) {
 	const examinationTimetable = await getCaseTimetableItems(response.locals.caseId);
 	const timetableItemsViewData = examinationTimetable?.items?.map(getTimetableRows) ?? [];
-	const republishStatus = examinationTimetable.items.some(
+	const republishStatus = examinationTimetable.items?.some(
```

Comment:
the "some" method returns a boolean. the "?" will return undefined rather than false. 
is that ok?

I think it'll work because nunjucks is not that smart to tell difference between false and undefined, but better to be sure.

Reply:
It passes testing on a case with and without exam items, but I can change this to something else if you prefer?